pkg/app/instances/docker.go

```diff
@@ -297,3 +312,51 @@ func (m *DockerInstanceManager) getContainerLabel(host string, key string) (stri
 	}
 	return value, nil
 }
+
+func (m *DockerInstanceManager) getDockerImageName(ctx context.Context) (string, error) {
+	listRes, err := m.Client.ImageList(ctx, image.ListOptions{})
+	if err != nil {
+		return "", fmt.Errorf("Failed to list docker images: %w", err)
+	}
+	for _, candidateImage := range m.Config.Docker.DockerImageNames {
+		for _, existingImage := range listRes {
+			for _, existingTag := range existingImage.RepoTags {
+				if candidateImage == existingTag {
+					return candidateImage, nil
+				}
+			}
+		}
+	}
+
+	var chosenImage string
```

Comment:
This logic to determine the fastest source is unreliable as you're making just one request which is not enough to describe latency, we would need a bigger sample. Let's have a configuration value for this and let the user change it depending on its own deployment.

Reply:
I had discussion about this with @ikicha, and decided to remove this logic and not bringing new configuration field `DockerImageNames`. `conf.toml` would be kept as current, and deployment of `conf.toml` to Artifact Registry would be happened by modifying the value of `conf.toml`.

Reply:
Removed that logic.